WordPressShared/Core/Views/WPNoResultsView.m

```diff
@@ -186,6 +190,20 @@ - (UIImage *)newButtonBackgroundImage {
     return [mainImage resizableImageWithCapInsets:capInsets];
 }
 
+- (NSAttributedString *)applyMessageStylesToAttributedString:(NSAttributedString *)attributedString
+{
+    NSRange fullTextRange = NSMakeRange(0, attributedString.string.length);
+    NSMutableAttributedString *mutableAttributedText = [attributedString mutableCopy];
+
+    [mutableAttributedText addAttribute:NSFontAttributeName value:self.messageTextView.font range:fullTextRange];
+    [mutableAttributedText addAttribute:NSForegroundColorAttributeName value:self.messageTextView.textColor range:fullTextRange];
+
+    NSMutableParagraphStyle *paragraphStyle = [NSMutableParagraphStyle new];
+    paragraphStyle.alignment = self.messageTextView.textAlignment;
+    [mutableAttributedText addAttribute:NSParagraphStyleAttributeName value:paragraphStyle range:fullTextRange];
+
+    return mutableAttributedText;
+}
```

Comment:
I've got doubts about this:

- The TextView's font and textColor are applied to the full attributedString text range
- Same for the TextAlignment

Bottom line: this method (kinda) defeats the purpose of having an attributedString property, since... no matter what font or color is passed along, it'll get overwritten.

May i ask you what's the use case / requirement that prompted this one?